packages/tpetra/core/src/Tpetra_Details_makeColMap_def.hpp

```diff
@@ -627,12 +627,16 @@ makeColMap (Teuchos::RCP<const Tpetra::Map<LO, GO, NT>>& colMap,
   //Pull down the sizes
   GO numLocalColGIDs = 0;
   GO numRemoteColGIDs = 0;
+  // DEEP_COPY REVIEW - DEVICE-TO-HOST
   Kokkos::deep_copy(numLocalColGIDs, numLocals);
+  // DEEP_COPY REVIEW - DEVICE-TO-HOST
   Kokkos::deep_copy(numRemoteColGIDs, numRemotes);
   //Pull down the remote lists
   auto localsHost = Kokkos::create_mirror_view(localGIDView);
   auto remotesHost = Kokkos::create_mirror_view(remoteGIDView);
+  // DEEP_COPY REVIEW - DEVICE-TO-HOST
```

Comment:
Yup.  This is a real, device to host copy of an array.  This goop could be replaced by a create_mirror_view_and_copy.  Though I we did that, we'd need to add a sync semantic check for it to Geoff's tests.